### PR TITLE
Feature/bbb 264 citizen app select your council

### PR DESCRIPTION
**Pull request by pauljroberts**

### CODE REVIEW

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/service/referencedata/ReferenceDataService.java [hunk 1]
```diff
@@ -68,73 +82,24 @@ private void initialise() {
...
   }
 
-  private List<ReferenceData> retrieveReferenceDataList(RefDataGroupEnum referenceDataGroup) {
+  public List<ReferenceData> retrieveReferenceDataList(RefDataGroupEnum referenceDataGroup) {
     initialise();
     return groupedReferenceDataList.get(referenceDataGroup.getGroupKey());
   }
 
-  private String retrieveReferenceDataDisplayValue(RefDataGroupEnum group, String key) {
+  public LocalAuthorityRefData lookupLaForLcCode(String localCouncilShortCode) {
```
[the remark this answers is not in the document]
Comment (pauljroberts) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/service/referencedata/ReferenceDataService.java]: Renamed.

File: src/main/resources/templates/applicant.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
         </div>
     </div>
-</main>
+</main>d
```
[the remark this answers is not in the document]
Comment (pauljroberts) [reply src/main/resources/templates/applicant.html]: Done.

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/service/referencedata/ReferenceDataService.java [hunk 3]
```diff
@@ -59,6 +65,14 @@ private void init() {
                       .collect(
                           Collectors.toMap(
                               ReferenceData::getShortCode, ReferenceData::getDescription))));
+
```
[the remark this answers is not in the document]
Comment (pauljroberts) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/service/referencedata/ReferenceDataService.java]: They ain't provided yet.

File: src/test/java/uk/gov/dft/bluebadge/webapp/citizen/service/ReferenceDataServiceTest.java [hunk 4]
```diff
@@ -88,101 +82,31 @@ public void whenExceptionFromAPI_thenLoadedSetBackToFalse() {
...
-    assertThat(gender).extracting("groupShortCode").containsOnly(GENDER.getGroupKey());
-    assertThat(gender).extracting("shortCode").containsOnly("MALE");
-  }
-
-  @Test
-  public void retrieveApplicationChannels_ShouldReturnApplicationChannels() {
-    List<ReferenceData> applicationChannels = referenceDataService.retrieveApplicationChannels();
-    assertThat(applicationChannels)
+  public void retrieveLocalCouncil_ShouldReturnLocalAuthorities() {
+    List<ReferenceData> las =
```
[the remark this answers is not in the document]
Comment (pauljroberts) [reply src/test/java/uk/gov/dft/bluebadge/webapp/citizen/service/ReferenceDataServiceTest.java]: Done.

File: acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/steps/SiteSteps.java [hunk 5]
```diff
@@ -232,4 +232,9 @@ public void iCanClickButton(String uiPath) throws Throwable {
   public void whenItypeTextForFieldUiPath(String text, String fieldId) throws Throwable {
     sitePage.findPageElementById(fieldId).sendKeys(text);
   }
+
+  @And("^I select \"([^\"]*)\" from autosuggest council list$")
```
Comment (stephen-bealine) [inline acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/steps/SiteSteps.java]: This shouldn't sit within Site Steps. Very specific to a page. 
Don't think it needs changing now. But think the steps need a bit of refactoring

Comment (sampathmahavithana) [reply acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/steps/SiteSteps.java]: Sure. I will move this to a specific page objects including few others in the next ticket. Thanks

**Pull request merged**